### ISSUE TITLE
Deleting last note of tag send you to top of tag tree (All Notes).

### ISSUE THREAD

**Issue by Armagedoom** [notable/notable#1172]
Currently, when deleting the last note of a tag, the tag selection goes to the top of the tag tree (All Notes).

But it should just go to the previous tag to the one just been deleted.

The reason behind this suggestion is, you are working on your tags and because they are close ones, you are probably going to keep working in that area rather than wanting to go to "All notes" after deleting that one tag.

**Comment by fabiospampinato** [comment]
> But it should just go to the previous tag to the one just been deleted.

I'm not so sure about that 🤔 

Currently in v1.9 when a situation like that happens there will just be no tags selected, which I guess should be ok for your use case too because the list basically doesn't scroll to another place, which I think is fundamentally the issue here. I'm not sure if this is better than moving to the "All Notes" section in general though 🤔 I guess let's keep it this way for now and see how it goes.

**Comment by Armagedoom** [comment]
Yeah, good for the suggested use case. The real problem was with jumping away from the current tag view.
